test/sagemaker_tests/autogluon/inference/requirements.txt

```diff
@@ -5,7 +5,7 @@ protobuf>=3.20,<3.21
 sagemaker>=2.31,<3
 smdebug>=1
 coverage==4.5.3
-docker-compose==1.25.4
+docker-compose
```

Comment:
Do we know why this was pinned in the first place?

Reply:
I think it was pinned because it was always pinned:
- https://github.com/aws/deep-learning-containers/pull/42
- https://github.com/aws/deep-learning-containers/pull/41